### DISCUSSION TITLE
Passband Luminosities and Absolute Measurements of Kepler Binaries

### DISCUSSION THREAD

**Discussion by podesse** [phoebe-project/phoebe2#430] [Questions]
Hello,

I'm working to fit binary models to Kepler data, and I've hit a bit of a snag trying to obtain absolute parameter values. I already have initial fits for these systems, and have been performing MCMC analysis on the photometric data reliably. However, my current program interpolates passband luminosity every time I call run_compute() – I have 'pblum_mode@lc@dataset' set to 'dataset-scaled' – so the luminosities of my resulting models are degenerate. I have found parallax measurements of my stars in Gaia so I was hoping to use some distance measurements to obtain observed synthetic flux in physical units. My current issue is that I don't quite understand how PHOEBE treats passband luminosities. Assuming I can convert the Kepler magnitude data from the KIC properly into physical units (ie. in W/m^2), and taking the distance measurements from Gaia for my models, what passband luminosity options and values should I use so that my synthetic observables produce fluxes in physical units that are comparable to the photometric data?

For example: my primary and secondary radius parameters appear to be correlated. I have a good constraint on the ratio of the primary to secondary radius for my systems, but can't obtain absolute parameter values for either since my MCMC distributions are non-gaussian. I realize that PHOEBE 2.3 has an option to propagate an MCMC distribution through the forward model, however I'm currently still running PHOEBE 2.2. As I understand it, with distance measurements I _should_ be able to obtain estimates of the radius of each individual star (knowing the distance, luminosity, and observed flux of the system would specify the radius and temperatures of the two stars). I've just not quite figured out how PHOEBE handles these particular variables.

Thank you!
Padraic

**Comment by kecnry** [comment]
Hi Padraic,

As long as your fluxes are calibrated and in real units, then you can set pblum_mode to 'absolute' and either fit or provide pblum (now in real luminosity units), distance (likely further away than the default of 1 meter which along with pblum of 4pi gave a flux contribution of roughly 1 from each star), and l3 (if applicable).  These three parameters will essentially handle scaling the output fluxes.  Without any external knowledge, pblum and distance will be severely degenerate, of course, but here you have some constraints from the distance to break that.

PHOEBE does not assume main-sequence (or otherwise) stars.  This means that the radii, temperatures, masses, and luminosities are all independent.  Since you're using 2.2, it sounds like you wrote your own MCMC wrapper, so I imagine you could just manually "link" the radii and luminosities under some assumed relationship when setting the values for each sample.  If you update to 2.3, you could do the same with a [custom constraint](http://phoebe-project.org/docs/2.3/tutorials/constraints_custom).  Formalizing this with an optional built-in constraint is on our [fitting wishlist](https://github.com/phoebe-project/phoebe2/discussions/407#discussioncomment-335421), so I'd be curious to hear your feedback on what would be most useful along those lines in the future.

Unfortunately we don't currently have any examples that do all of this, although it would be a good idea to add one (if you're interested in contributing an example once you have everything working, let me know, that would be great!).  That said, here are the relevant docs that may be of help:

* [pblum](http://phoebe-project.org/docs/2.2/tutorials/pblum)
* [third light](http://phoebe-project.org/docs/2.2/tutorials/l3)
* [distance](http://phoebe-project.org/docs/2.2/tutorials/distance)
